README.md

```diff
@@ -218,6 +218,7 @@ aem {
         satisfyBundlePath = 
         satisfyBundleProperties = { bundle -> [:] }
         satisfyGroupName = "*"
+        trustAllCertificates = false
```

Comment:
this parameter semantically is nearest to `deployConnectionTimeout` , so I think that better if it will follow convention `deployXXX` for shared parameters deployment related.

what do you think about name `deployConnectionSslUntrusted` ? please order that property near to that one (that class is a part of documentation, so I am trying to group parameters which are semantically close to each other).